src/containers/ToDoList/ToDoList.jsx

```diff
@@ -81,22 +102,54 @@ export const ToDoList = () => {
         ))}
 
         <StyledWrapperSelect>
-          <StyledParagraph>Items left</StyledParagraph>
+          <StyledParagraph>
+            {tasks.filter((task) => task.checked).length} items left
+          </StyledParagraph>
           <StyledWrapperSelectButtonGroup>
-            <button>All</button>
-            <button>Active</button>
-            <button>Completed</button>
+            <StyledFilterButton
+              isActive={filterTask === "all"}
+              onClick={() => setFilterTask("all")}
+            >
+              All
+            </StyledFilterButton>
+            <StyledFilterButton
+              isActive={filterTask === "active"}
+              onClick={() => setFilterTask("active")}
+            >
+              Active
+            </StyledFilterButton>
+            <StyledFilterButton
+              isActive={filterTask === "completed"}
+              onClick={() => setFilterTask("completed")}
+            >
+              Completed
+            </StyledFilterButton>
           </StyledWrapperSelectButtonGroup>
-          <StyledButtonClearCompletedTask>
+          <StyledButtonClearCompletedTask onClick={handleClearCompletedTasks}>
             Clear Completed
           </StyledButtonClearCompletedTask>
         </StyledWrapperSelect>
       </StyledWrapper>
 
       <StyledWrapperSelectButtonGroupSeparated>
-        <button>All</button>
-        <button>Active</button>
-        <button>Completed</button>
+        <StyledFilterButton
+          isActive={filterTask === "all"}
+          onClick={() => setFilterTask("all")}
+        >
+          All
+        </StyledFilterButton>
+        <StyledFilterButton
```

Comment:
Are these buttons here as well because of the RWD? If yes, and everything between them is the same, in order to not have to repeat the code, you'd wanna wrap them in a separate component function (doesn't even have to be a separate file) and then reuse this component in those 2 places.